example/classifier/Makefile.am

```diff
@@ -5,10 +5,8 @@ bin_PROGRAMS = odp_classifier
 odp_classifier_SOURCES = odp_classifier.c
 
 if test_example
-if ODP_PKTIO_PCAP
 TESTS = odp_classifier_run.sh
 endif
-endif
```

Comment:
Each platform which supports pcap is supposed to define ODP_PKTIO_PCAP. Currently both linux-generic and linux-dpdk do this. If your platform supports pcap, you should also define ODP_PKTIO_PCAP.

Reply:
ODP_PKTIO_PCAP actually is just linux-generic platform's feature to support pcap based PKTIO where source is from PCAP. This I think shouldn't be a generic feature as other platforms would never support such HW device which pick from pcap ? For our platform, we feed pcap packets from our internal LBK HW interface.

Reply:
Are you referring to pcap live capture, which pcap pktio does not use? Pcap pktio only uses pcap to read and write capture files. So nothing to do with any hardware, and not specific to linux-generic, since libpcap (or equivalent) is pretty widely available.

Looking a bit closer, ODP_PKTIO_PCAP just depends on whether libpcap is available, so there shouldn't be any need to touch it.